lib/build.js

```diff
@@ -236,19 +240,24 @@ module.exports = {
         });
 
         _.forEach(targets, function (target, targetName) {
-          if (!target.keymap) {
-            target.dispose = _.noop;
-            return;
+          var keymapDispose = null;
+          var commandName = 'build-' + targetName;
+          // there may be only one : in commandName
```

Comment:
Why can there only be one `:` ? It works well with multiple now.

I'm not sure why you need to replace anything at all from the command name.